modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -328,7 +333,8 @@ public CompletableFuture<Void> startAsync() {
                     ApplyTask::new,
                     opts.getStripes(),
                     false,
-                    false
+                    false,
+                    opts.getRaftMetrics().disruptorMetrics("JRaft-FSMCaller-Disruptor")
```

Comment:
Not sure, that I understand the idea - why we set here "JRaft-FSMCaller-Disruptor" name and then convert it to "jraft.fsmcaller.disruptor" immediately? Maybe use the "jraft.fsmcaller.disruptor" here directly?